source/develop/dev-process/working-with-gerrit.html.md

```diff
@@ -101,80 +101,82 @@ Or alternatively without need of authentication:
       git commit --amend -s
        
 
-### Configure git personal settings
+### Configuring Git Personal Settings
 
       git config --global user.name "John Doe"
       git config --global user.email johndoe@example.com
        
 
-### Configure the commit template
+### Configuring the Commit Template
 
       cd ovirt-engine
       git config commit.template config/engine-commit-template.txt
        
 
-### Configure git commit spell check, syntax highlighting and maxwidth
+### Configuring the Git Commit Spell Check, Syntax Highlighting and Maxwidth
 
       # edit ~/.vimrc
       syntax on
       filetype plugin indent on
       autocmd Filetype gitcommit spell textwidth=72
        
 
-### Rebase
+### Performing Rebase
 
       git fetch -v
       git rebase origin/master
        
 
-### Push your patch for review
+## Pushing a Patch for Review
 
-### push/update the patch as a DRAFT
+### Pushing or Updating a Patch as a Draft
 
       git push gerrit.ovirt.org:ovirt-engine HEAD:refs/drafts/master
        
 
-or, assuming you remote repo is origin you could also
+Alternatively, and assuming your remote repository is 'origin', enter:
 
       git push origin HEAD:refs/drafts/master
        
 
-### push the patch after it is published
+### Pushing a Patch After It Is Published:
 
       git push gerrit.ovirt.org:ovirt-engine HEAD:refs/for/master
        
 
-or, assuming you remote repo is origin you could also
+Alternatively, assuming your remote repository is 'origin', enter:
 
       git push origin HEAD:refs/for/master
        
 
-### the same using git review plugin
+### Pushing or Updating a Draft Using the Git Review Plugin
 
-pushing/updating a draft:
+To push or update a draft, enter:
 
       git review -D -r origin master
        
 
-pushing/updating a published patch:
+To push or update a published patch:
 
       git review -r origin master
        
 
-## Track patch review process
+## The Track Patch Review Process
+
+### A Summary of the Process
 
-The patch life cycle process comprises:
+The patch life cycle process comprises of the following steps:
 
-1.  patch checked
-    -   draft patch for early reviews
-    -   verification (Verified+1)
-    -   maintainer review approval (Code-Review+2)
-    -   publishing [triggers CI]
+1.  ‎The patch is checked
+    -   Create a draft patch for early reviews
```

Comment:
assign specific people to review it, if none were added automatically. Otherwise, don't use Draft.